modules/images-other-jenkins-oauth-auth.adoc

```diff
@@ -18,12 +18,28 @@ Jenkins supports both browser and non-browser access.
 
 Valid users are automatically added to the Jenkins authorization matrix at log
 in, where {product-title} `Roles` dictate the specific Jenkins permissions the
-users have.
+users have.  The `Roles` used by default are the `admin`, `edit`, and `view` 
+`Roles` in the `Project` or namespace that Jenkins is running in.
```

Comment:
s/`Roles` in the `Project`/in the `Project`

Reply:
No don't think so, but I agree what I had was not great either.  Will have a new version of this in the upcoming update.